### ISSUE TITLE
Make some telemetry fields 'persistent'

### ISSUE THREAD

**Issue by darksidelemm** [projecthorus/sondehub-tracker#300]
Currently the per-sonde info area is updated packet-by-packet. This means that if only some stations are uploading a particular field of interest (e.g. the new rs41_mainboard data), then it might only appear briefly, then get wiped out when another station uploads a packet without that data.

Suggest having a list of persistent variables, which get updated, but don't get removed.

**Comment by Mauri740** [comment]
Yes, the receiving stations section is very difficult to read. Its continuous movement due to the updating of telemetry makes reading very complicated.

**Comment by darksidelemm** [comment]
Some thoughts on this.
- Instead of just updating the vehicle data block and re-generating the vehicle info from that, maintain a separate 'latest_data' store, which is updated based on new data. For each field, include an 'ageout' value, initially set to something like 100. With every update, if the data hasnt been updated, decrement the ageout value. Once it reaches 0, remove it.
- Consider having data fields which havent been updated recently (based on the ageout value), be shown in a different shade, perhaps grey. This will hopefully distinguish current and old data.

**Comment by Mauri740** [comment]
it would be a good thing to do. In this way the list of receivers would be updated with a more logical system. To make the list of receivers static and no longer in continuous movement caused by different telemetry data, space could be created by deleting unimportant telemetry data such as the sonde manufacturer parameter.

**Comment by darksidelemm** [comment]
Notes to self...
- the field data to be displayed is in the .data variable within each position.
- To get the intended behaviour, we will need to maintain a separate store, which we add fields to, and just update fields that already exist. We also need a separate store with the ages of each field.
- The data is formatted into the table shown on the screen in the habitat_data function

**Comment by darksidelemm** [comment]
Some changes getting rolled out into testing.
- Data fields are now persistent, and get updated if the incoming data is newer than the old data for that field.
- If a data field is older than 20 seconds from the current latest data, then the field name (under the field value) is appended with (XXs old). Need to think more about how this should be represented.

Unsure if my current source for the 'current' time is good enough - Need to see how this handles out-of-order packet arrivals.

**Comment by darksidelemm** [comment]
Testing tracker is here: https://testing.v2.sondehub.org/

**Comment by Mauri740** [comment]
I think this is better! Great job! Thank You

**Comment by TheSkorm** [comment]
@darksidelemm any more needed on this?
